src/bench/crypto_hash.cpp

```diff
@@ -199,6 +200,98 @@ static void SipHash_32b(benchmark::Bench& bench)
     });
 }
 
+static void SaltedOutpointHasherBenchmark_hash(benchmark::Bench& bench)
+{
+    FastRandomContext rng(true);
+    constexpr size_t size = 1000;
+
+    std::vector<COutPoint> outpoints(size);
+    for (auto& outpoint : outpoints) {
+        outpoint = {Txid::FromUint256(rng.rand256()), rng.rand32()};
+    }
+
+    const SaltedOutpointHasher hasher;
+    bench.batch(size).run([&] {
+        size_t result = 0;
+        for (const auto& outpoint : outpoints) {
+            result ^= hasher(outpoint);
```

Comment:
In "bench: Add COutPoint and SaltedOutpointHasher benchmarks" 6ce13a33c6901d820467b9c24a03ca3861d91640

Why are you xoring the hashes?

Reply:
Just to make sure they're not optimized away - ankerl::nanobench::doNotOptimizeAway is likely more costly than a simple xor.
Do you have a better idea?